processing/sensorhub-process-opencv/build.gradle

```diff
@@ -17,9 +17,14 @@ dependencies {
 osgi {
   manifest {
     attributes('Bundle-Vendor': 'Sensia Software LLC')
-    attributes('Bundle-Activator': 'org.sensorhub.impl.process.opencv')
+    attributes('Bundle-Activator': 'org.sensorhub.impl.process.opencv.Activator')
     attributes('-fixupmessages': 'Classes found in the wrong directory; is:=ignore')
-    attributes('-exportcontents': 'org.bytedeco.opencv.*, org.opencv.*')
+    attributes('-exportcontents': 'org.bytedeco.opencv, org.opencv,org.sensorhub.impl.process.opencv')
```

Comment:
Why remove the wildcards?

Reply:
I think the wildcard may not work for 'exportcontents', only work with Import-Package

Reply:
But with this setting, is everything under `org.bytedeco.opencv` exported recursively in the final MANIFEST?

Reply:
After testing, I think there are more packages exported using wildcard, so I've restored them: 7a4463b770a915e60fde6d6cc0871d52325c6f6a